docs/chained-sessions.md

```diff
@@ -0,0 +1,161 @@
+		result.Disclosed[0][0].Identifier.String() != "irma-demo.gemeente.personalData.fullname" ||
+		result.Disclosed[0][0].RawValue == nil {
+		writeError(w, "received invalid session result")
+		return
+	}
+
+	// Construct issuance request to respond with,
+	// containing the attribute from the session result
+	issuanceRequest := irma.NewIssuanceRequest([]*irma.CredentialRequest{{
+		CredentialTypeID: irma.NewCredentialTypeIdentifier("irma-demo.IRMATube.member"),
+		Attributes: map[string]string{
+			"fullname": *result.Disclosed[0][0].RawValue,
+			"type":     "premium",
+			"id":       "123456",
+		}}},
+	)
+
+	// Marshal and write our issuance request
+	bts, err = json.Marshal(issuanceRequest)
+	if err != nil {
+		writeError(w, err.Error())
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	if _, err = w.Write(bts); err != nil {
+		fmt.Println(err.Error())
+	}
+}
+
+func writeError(w http.ResponseWriter, msg string) {
+	w.WriteHeader(500)
+	w.Write([]byte(msg))
+}
+```
+
+The session chain can then be started by sending the following session request to our IRMA server.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Extended session request (JSON)-->
+```jsonc
+{
+    "nextSession": {
+        "url": "https://example.com"
+    },
+    "request": {
+        "@context": "https://irma.app/ld/request/disclosure/v2",
+        "disclose": [[[ "irma-demo.gemeente.personalData.fullname" ]]]
+    }
+}
+```
+<!--Extended session request (Go)-->
+```go
+irma.ServiceProviderRequest{
+	RequestorBaseRequest: irma.RequestorBaseRequest{
+		NextSession: &irma.NextSessionData{
+			URL: "https://example.com",
+		},
+	},
+	Request: irma.NewDisclosureRequest(
+		irma.NewAttributeTypeIdentifier("irma-demo.gemeente.personalData.fullname"),
+	),
+}
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+After the user discloses the `irma-demo.gemeente.personalData.fullname` attribute, the IRMA server POSTs it to `https://example.com`. Our Go server responds to the IRMA server with the `issuanceRequest` found near the end of the program. Next, the IRMA server immediately starts this issuance session with the user's IRMA app, resulting in the screenshots shown on top of this page.
+
+## Differences with combined issuance-disclosure requests
+
+[Combined issuance-disclosure requests](session-requests.md#issuance-requests), i.e. issuance requests with a nonempty `disclose` field requesting attributes to be disclosed, is another way of first requesting and then issuing attributes from/to an IRMA app user within a single flow. Thus, this is very similar to a session chain consisting of first a disclosure request and then an issuance request. Contrary to session chains, however, this flow is started using a single session request. Thus, using such session requests it is impossible for the issued attributes to depend on the disclosed attributes, because at the time the session request is composed the value of the disclosed attributes are not yet known.
```

Comment:
I had some difficulties parsing the text of this block. I experienced the `Contrary to session chains, however, this flow...` sentence difficult to read. I'd more expect something like `However, contrary to session chains, this flow ...`. 

Furthermore, in the sentence after that, the reference in `such session requests` is a bit sloppy in my opinion, because you also mention chained sessions in the sentence before. It makes the sentence a bit complicated. Maybe you can say `Thus, when using combined issuance-disclosure requests, the issued attributes cannot depend  on the disclosed attributes.`